tests/commands/completion/test_completions_command.py

```diff
@@ -48,10 +49,7 @@ def test_bash(mocker: MockerFixture) -> None:
     tester = CommandTester(command)
     tester.execute("bash")
 
-    with open(
-        os.path.join(os.path.dirname(__file__), "fixtures", "bash.txt"),
-        encoding="utf-8",
-    ) as f:
+    with (FIXTURES_PATH / "bash.txt").open(encoding="utf-8") as f:
         expected = f.read()
```

Comment:
I think you could also use `Path.read_text` here :+1:
(also applies to L74 and L96)